bridge-stp.in

```diff
@@ -52,7 +58,7 @@ case $2 in
         if [ -e "$conf_file" ]; then
             . "$conf_file" || exit
         fi
-        checkpid $pid_file || exit 1
+        checkpid || exit 1
```

Comment:
personally, i like having the `pid_file` as argument here
but this is more of a preference